neurom/apps/tests/test_morph_stats.py

```diff
@@ -140,6 +144,63 @@ def test_extract_stats_single_neuron():
             assert_almost_equal(res[k][kk], REF_OUT[k][kk], places=3)
 
 
+def test_extract_dataframe():
+    # Vanilla test
+    nrns = nm.load_neurons([os.path.join(SWC_PATH, name)
+                            for name in ['Neuron.swc', 'simple.swc']])
+    actual = ms.extract_dataframe(nrns, REF_CONFIG)
+    expected = pd.read_csv(os.path.join(DATA_PATH, 'extracted-stats.csv'), index_col=0)
+    assert_frame_equal(actual, expected)
+
+    # Test with a single neuron in the population
+    nrns = nm.load_neurons(os.path.join(SWC_PATH, 'Neuron.swc'))
+    actual = ms.extract_dataframe(nrns, REF_CONFIG)
+    assert_frame_equal(actual, expected[expected.name == 'Neuron'], check_dtype=False)
+
+    # Test with a config with the 'neuron' key
+    nrns = nm.load_neurons([os.path.join(SWC_PATH, name)
+                            for name in ['Neuron.swc', 'simple.swc']])
+    config = {'neurite': {'total_length_per_neurite': ['total']},
+              'neurite_type': ['AXON', 'APICAL_DENDRITE', 'BASAL_DENDRITE']}
+    actual = ms.extract_dataframe(nrns, config)
+    expected = pd.DataFrame(
```

Comment:
Can we reuse `expected` from before? Something like `expected[['total_total_length_per_neurite']]`?